gen/input/src/main/scala/test/StarWarsQuery3.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+// format: off
+/*
+  rules = [GraphQLGen]
+  GraphQLGen.schemaDirs = ["gen/input/src/main/resources/graphql/schemas"]
+ */
+package test
+
+import clue.GraphQLOperation
+import clue.annotation.GraphQL
+import test.StarWars
+
+@GraphQL
+trait StarWarsQuery3 extends GraphQLOperation[StarWars] {
+  override val document: String = s"""
+        query ($$charId: ID!) {
+          character(id: $$charId) {
+            id
+            name
+            ... on Human {
+              homePlanet
+            }
+            friends $StarWarsSubquery
```

Comment:
Interpolation works like this.

Reply:
Oh, is this just regular string interpolation? How does this work behind the scenes?

Reply:
Yup, at the moment this is just cheating with ordinary string interpolation. It's calling `StarWarsSubquery.toString`.

Probably what we want to do long-term is introduce a `gql"..."` interpolator that handles these specially.